kallisto-compose.yaml

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2022 by the Lucyfer Contributors
+
+# vim: set fileencoding=utf-8:
+
+
+version: '3'
+
+services:
+    kallisto_service:
+        container_name: 'kallisto'
```

Comment:
Specifying the container_name here prevents multiple projects to exist side by side.  For docker-compose it's better to specify service names and let docker-compose uniquely generate the container name.  docker compose knows how to build hostname resolution between containers based on the service name so there's not really a practical reason to specify the container_name here

Reply:
This isn't a generic Docker Compose file; it's specific to starting Lucyfer + Kallisto only.  For generic/3rd-party/non-standard Docker Compose files use the `LUCYFER_COMPOSE` environment variable.  That forces `lucy` to override the configured container names and services.